build_info/duktape.control

```diff
@@ -0,0 +1,34 @@
+Package: duktape
+Version: @DEB_DUKTAPE_V@
+Architecture: @DEB_ARCH@
+Maintainer: @DEB_MAINTAINER@
```

Comment:
Add the author of the package on all your control files below the Maintainer field.

Reply:
Also, there is no author for this package
Maybe if you check Debian first before posting